src/anypoint-input-styles.js

```diff
@@ -200,6 +200,10 @@ export default css`
     box-sizing: border-box;
   }
 
+  :host([compatibility]) .label.compatibility {
+    top: -22px;
```

Comment:
What does this affect?

Reply:
It affects input label when theme is Anypoint.